### PR TITLE
Add script for the becoming a contributor section.

### PR DESCRIPTION
**Pull request by MaineC**
Essentially this translates the speaker notes of the slides for this section into a script.

### CODE REVIEW

**Comment by rrrutledge** [comment]
Awesome, @MaineC!  I will take a look at this, likely over the weekend.

**Comment by rrrutledge** [comment]
This addresses https://github.com/InnerSourceCommons/InnerSourceLearningPath/issues/158

File: contributor/02-becoming-a-contributor-script.md [hunk 1]
```diff
@@ -10,3 +10,70 @@ How do some people seem to find opportunity to contribute to projects everywhere
 
 ## Script
 
+X: Welcome back to our video series on the contributor role in InnerSource. In
+this episode you are going to learn more on what it takes to become a successful
+contributor in the InnerSource project of your choice. We'll be checking out
+what a sharing mindset has to do with contributing, what the benefits of sharing
+solutions are and which types of contributions InnerSource projects are
+typically looking for.
+
```
Comment (lenucksi) [inline contributor/02-becoming-a-contributor-script.md]:
```suggestion

-> Show gaining by sharing slide. 
```

File: contributor/02-becoming-a-contributor-script.md [hunk 2]
```diff
@@ -10,3 +10,70 @@ How do some people seem to find opportunity to contribute to projects everywhere
...
+X: Imagine you are implementing a new feature for your product. Before jumping
+right into the implementation - slow down for a bit: Is this a general issue
+that could benefit from a solution that multiple teams work on? Do other teams
+face the same challenge because they operate in the same business domain?  Is
+this functionality orthogonal to your business domain?
+
+X: If any of this is the case - look beyond your own team: Is there a shared
+solution that you can rely on? Should there be one?
+
```
Comment (lenucksi) [inline contributor/02-becoming-a-contributor-script.md]:
```suggestion

-> Show dealing with dependencies slide 1
```

File: contributor/02-becoming-a-contributor-script.md [hunk 3]
```diff
@@ -10,3 +10,70 @@ How do some people seem to find opportunity to contribute to projects everywhere
...
+X: So you've found that the functionality that you need already exists within
+your company. On a technical level, the solution is easy: Introduce a dependency
+in your project, go ahead and re-use what others did before. What looks easy on
```
Comment (lenucksi) [inline contributor/02-becoming-a-contributor-script.md]:
```suggestion
in your project, go ahead and re-use what others did before. 

-> -> Show dealing with dependencies slide 2

What looks easy on
```

File: contributor/02-becoming-a-contributor-script.md [hunk 4]
```diff
@@ -10,3 +10,70 @@ How do some people seem to find opportunity to contribute to projects everywhere
...
+first sight becomes more complicated when we remember that all of those
+components are being built by humans - colleagues, working in different teams,
+potentially against different roadmaps, potentially using different design
+patterns than your own team. Introducing dependencies light-heartedly comes at a
+cost: The features you need, may not be on the roadmap of the team working on
+that component. The component may lack some minor modification that you need in
+order to meaningfully use it. At the end of the day you make your success
+dependent on another team.
+
+X: The typical solution is to cut dependencies whereever possible. This means
```
Comment (lenucksi) [inline contributor/02-becoming-a-contributor-script.md]:
```suggestion

-> -> Show dealing with dependencies slide 3

X: The typical solution is to cut dependencies whereever possible. This means
```

File: contributor/02-becoming-a-contributor-script.md [hunk 5]
```diff
@@ -10,3 +10,70 @@ How do some people seem to find opportunity to contribute to projects everywhere
...
+that you can move independent and faster. It also means that you will duplicate
+effort that others already invested. InnerSource gives you a path in between the
```
Comment (lenucksi) [inline contributor/02-becoming-a-contributor-script.md]:
```suggestion
effort that others already invested. 

-> -> Show dealing with dependencies slide 4

InnerSource gives you a path in between the
```

File: contributor/02-becoming-a-contributor-script.md [hunk 6]
```diff
@@ -10,3 +10,70 @@ How do some people seem to find opportunity to contribute to projects everywhere
...
+
+X: Contributing those changes to the other team's codebase means that they will
+take over maintanance for you going forward reducing the work that you will have
+to do.
+
+As a good contributor you should be able to make a call for when re-using a
+component provides a benefit to your team in terms of collaboration with others
+to move faster and reduce waste. 
+
+X: So - it's called InnerSource - that means, this is just about source-code,
```
Comment (lenucksi) [inline contributor/02-becoming-a-contributor-script.md]:
```suggestion

-> -> Show more than source code slide

X: So - it's called InnerSource - that means, this is just about source-code,
```

**Pull request merged**